heareval/predictions/runner.py

```diff
@@ -74,6 +82,21 @@ def runner(
         if not task_path.is_dir():
             raise ValueError(f"{task_path} should be a directory")
 
+        # Get the output directory name
```

Comment:
Uhm.... you should be branching off `leader` branch. Otherwise there is no way to re-use my code that checks if prediction has already been done and we can skip it for this model + prediction

Reply:
Yup. My bad. Updated.